book/website/reproducible-research/open/open-source.md

```diff
@@ -1,237 +1,21 @@
 (rr-open-source)=
-# Open Source Software
+# Open Source
 
-(rr-open-source-whatis)=
-## What Is Open Source Software?
+## Definition
 
-When a project is open-source [{term}`def<Open Source Software>`], anybody can view, use, modify, and distribute the project for any purpose.
-These permissions are enforced through an open-source licence.
-Open source is powerful because it lowers the barriers to adoption, allowing ideas to spread quickly.
-In its most basic form, open-sourcing your software means putting your code online where it can be viewed and reused by others.
+### Noun
 
-Many of the most widely used research software is open source.
-Perhaps the paradigmatic example is the scikit-learn Python package for machine learning (Pedregosa et al., 2011), which, in the space of just over five years, has attracted over 500 unique contributors, 20,000 individual code contributions, and 2,500 article citations.
-Producing a comparable package using a traditional closed-source approach would likely not be feasible.
-It would, at the very least, require a budget of tens of millions of dollars.
-While scikit-learn is an outlier, hundreds of other open-source packages that support much more domain-specific needs depend similarly on unsolicited community contributions; for example, the NIPY (neuroimaging in Python) group of projects in neuroimaging (Gorgolewski et al., 2016).
-Notably, such contributions not only result in new functionality from which the broader community can benefit, but also regularly provide their respective authors with greater community recognition, and lead to new project and employment opportunities.
+Practice of providing digital resources with their sources.
```

Comment:
I'm not sure if that definition is a noun.

[This](https://www.collinsdictionary.com/dictionary/english/open-source) could be a guide.

Reply:
You can find it both as [noun](https://en.wiktionary.org/wiki/open_source) and [adjective](https://en.wiktionary.org/wiki/open-source#English) on some dictionaries (example from Wiktionary).

**Example:** *Open source is a widespread phenomenon in IT.*

Reply:
Yes, I'm not saying that it isn't a noun. It's just that that definition in particular didn't sound like a noun to me, more like an adjective (or verb?). It is very similar to one of the wiktionary noun definitions though.

Reply:
>  very similar to one of the wiktionary noun definitions though

It's finally a reuse of Wiktionary definitions for both (openly licensed ideas ?). I was looking for a dictionary and how they structure definitions, I ended up on this. It looks efficient to convey the idea, it's just a tweak on them. It seems to have the basic components of the concept to stay simple. 

I didn't pay attention to the following on the [open source adjective](https://en.wiktionary.org/wiki/open-source#English) of the Wikitionary:
> 2. (by extension) Describing any [product](https://en.wiktionary.org/wiki/product#English) whose [composition](https://en.wiktionary.org/wiki/composition#English) or method of [manufacturing](https://en.wiktionary.org/wiki/manufacturing#English) is [public](https://en.wiktionary.org/wiki/public#English) [knowledge](https://en.wiktionary.org/wiki/knowledge#English) and not [proprietary](https://en.wiktionary.org/wiki/proprietary#English).

This is not a redefinition, this a remix of definitions !

Reply:
The verbal form is not one that I use, one that I've thought about and one that I'm comfortable with, it could be additional elements for the future. The idea is to have a simple base and not the right, exhaustive definition.

I think there is a choice to do between continuing to explain as it is today to follow the conventional probably wrong meaning next to an unreaded exposition of the debate or break backward compatibility and trying to bring the state-of-the-art understanding of the concept of open source.

A choice between following and maintaining popular beliefs or following the opinion of experts.